packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/selectedMessageName.st

```diff
@@ -0,0 +1,7 @@
+code browser
+selectedMessageName
+
+	self currentSelection
+	ifNil: [^ nil].
```

Comment:
This nil-check is a duplicate of the first lines in `IPTTreeModel >> selectedMessage`. Please move it to a function used in both places to avoid code duplication.